### DISCUSSION TITLE
EventSystem의 Interface를 사용할 수 있나요?

### DISCUSSION THREAD

**Discussion by Asipe42** [naverz/zepeto-studio-kor#1061] [5. Q&A]
EventSystem의 IPointerClickHandler와 같은 인터페이스를 상속해

해당 인터페이스의 함수들을 오버라이딩 해서 사용하고 싶은데.. 제페토에서 가능한가요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 아직까지 지원하지 않습니다. 
https://github.com/naverz/zepeto-studio-kor/discussions/632 이후로 관리하고 있는 이슈이나
다른 크리티컬한 이슈들도 많기에 지원까지 시간은 다소 소요되는점 양해 부탁 드립니다. 
감사합니다.
